Patterns/English/English-DateTime.yaml

```diff
@@ -92,7 +92,7 @@ OfPrepositionRegex: !simpleRegex
   def: (\bof\b)
 # Exclude cases that include the "Am/Pm" suffix
 TwoDigitYearRegex: !nestedRegex
-  def: \b(?<![$])(?<year>([0-24-9]\d))(?!(\s*((\:\d)|{AmDescRegex}|{PmDescRegex}|\.\d)))\b
+  def: \b(?<![$])(?<year>([0-9]\d))(?!(\s*((\:\d)|{AmDescRegex}|{PmDescRegex}|\.\d)))\b
```

Comment:
Doesn't this issue affect also other languages?

Reply:
yes, modified regex in other languages too.